### DISCUSSION TITLE
Why custom open_ai client crate?

### DISCUSSION THREAD

**Discussion by ifsheldon** [zed-industries/zed#16332] [Feature Requests]
Hi! I wonder why you choose to implement your own `open_ai` client crate? I took a look at the code, and the implementation is concise.

I am the maintainer of `async-openai-wasm` and I think it's more complete in terms of functionalities.  I think it's better to maintain one crate to keep up with the pace of OpenAI changing APIs instead of reinventing wheels, but I am definitely not implying that you should use my crate. This discussion is simply out of curiosity and I want to see what in `async-openai-wasm` can be improved.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
